### DISCUSSION TITLE
Loading Image from bytes

### DISCUSSION THREAD

**Discussion by msciancalepore98** [microsoft/onnxruntime-genai#663] [New features / APIs]
Hey, I am trying to integrate your phi-3-vision script into LitServe.

How can I use the already loaded image (bytes) and pass it to the processor? (I don't want to save it as local/tmp file)

Thanks

**Comment by baijumeswani** [comment]
Hi sorry for the late response. We do not have support for loading an image from bytes at this point.

We can enhance the usage for the next release `0.4.0`.

**Comment by elephantpanda** [comment]
Yes, this would be very welcome. Especially for things like getting an image from the webcam, or a screenshot. Would be nice to avoid unnecessary writing to the disk.

**Comment by kunal-vaishnavi** [comment]
Support for loading images from memory has been added [here](https://github.com/microsoft/onnxruntime-genai/pull/796). You can use the `og.Images.open_bytes` method to do this.

```python
image = og.Images.open_bytes(bytes)
```

**Comment by dhhunter** [comment]
How do you load multiple images from byte arrays?
